src/main/scala-3/com/eed3si9n/eval/Eval.scala

```diff
@@ -295,9 +295,10 @@ object Eval:
     }
 
   def currentClasspath: Seq[Path] =
-    val cl = ClassLoader.getSystemClassLoader()
```

Comment:
They change a class behind SystemClassLoader in Java 9. https://stackoverflow.com/a/46724308/6176274